python_on_whales/docker_client.py

```diff
@@ -250,6 +251,11 @@ def login_ecr(
         ```
 
         You need botocore to run this function. Use `pip install botocore` to install it.
+
+        The `registry` parameter can be used to override the registry that is guessed from the authorization token
+        request's response.
+        It is especially useful if the aws account you use can access several repositories and you
```

Comment:
```suggestion
        The `registry` parameter can be used to override the registry that is guessed from the authorization token
        request's response.
        In other words: If the registry is `None` (the default) then it will be assumed that it's the ECR registry linked to the credentials provided.
        It is especially useful if the aws account you use can access several repositories and you
```